### DISCUSSION TITLE
Apple Silicon

### DISCUSSION THREAD

**Discussion by peterl-bmd** [orbitalquark/textadept#8] [General]
Greetings!

Just wondering if the macOS build is going to be updated to be a universal binary for the ARM based Apple Silicon macs?

Thanks!
- Peter

**Comment by orbitalquark** [comment]
Nope. Two things are needed for that:

1. GTK-OSX (the GUI toolkit Textadept uses) needs to be compiled/provided for the ARM arch.
2. There needs to be a way to cross-compile for ARM from Linux using Docker.

Textadept will need to be run using Rosetta for the forseeable future.
